packages/server/lib/socket-base.ts

```diff
@@ -67,12 +67,14 @@ export class SocketBase {
   private _isRunnerSocketConnected
   private _sendFocusBrowserMessage
 
+  protected inRunMode: boolean
   protected supportsRunEvents: boolean
   protected ended: boolean
   protected _io?: socketIo.SocketIOServer
   localBus: EventEmitter
 
   constructor (config: Record<string, any>) {
+    this.inRunMode = config.isTextTerminal
```

Comment:
Should this just be a getter instead of a duplicate property?

```js
get inRunMode () {
  return config.isTextTerminal
}
```

Reply:
The configuration isn't available on this class outside of initialization. It's lights pull out this specific property.

Reply:
Right, didn't realize that - no problem.